tests/data_import/test_ze_aliases.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from django.core.exceptions import ValidationError
+
+from data_import.models import RespondingAgencyAlias
+
+
+@pytest.mark.django_db
+def test_responding_agency_preferred_alias(responding_agency):
+    agency = responding_agency.build()
+
+    old_alias = RespondingAgencyAlias.objects.create(responding_agency=agency, name='a_rose', preferred=True)
+
+    new_alias = RespondingAgencyAlias.objects.create(responding_agency=agency, name='by_any_other_name', preferred=True)
+
+    old_alias_again = RespondingAgencyAlias.objects.get(id=old_alias.id)
+
+    if new_alias.preferred:
+        pass
+    if not old_alias_again.preferred:
+        pass
```

Comment:
What's happening here?

Reply:
When I was trying to set these assertions like so: 
```python
assert new_alias.preferred == True
assert old_alias_again.preferred == False
```

I was getting a couple of flake8 errors: 
```shell
/app/tests/data_import/test_ze_aliases.py:17:32: E712 comparison to True should be 'if cond is True:' or 'if cond:'
/app/tests/data_import/test_ze_aliases.py:18:38: E712 comparison to False should be 'if cond is False:' or 'if not cond:'
```
I saw pass had been used to get the test to pass, but your question is making me wonder if I misinterpreted that keyword. What's the best way to do this?

Reply:
In Python, [`pass` means no operation](https://docs.python.org/2.0/ref/pass.html). `flake8` is complaining about using double equals comparison for a Boolean value. To get rid of those errors, use `assert new_alias.preferred is True`, etc., as the message suggests. Make sense?